app/widgets/deployer-bar.js

```diff
@@ -307,6 +307,7 @@ YUI.add('deployer-bar', function(Y) {
 
       if (container && container.get('parentNode')) {
         container.one('.panel.summary section').setHTML(this.summaryTemplate({
+          autoPlaceDefault: localStorage.getItem('auto-place-default'),
```

Comment:
maybe a comment here referencing the hacking config UX since they're a bit disconnected please.